hawc/apps/hawc_admin/urls.py

```diff
@@ -0,0 +1,98 @@
+from typing import Any, List
+
+from django.conf import settings
+from django.contrib import admin
+from django.urls import include, path
+from rest_framework import permissions
+from rest_framework.routers import DefaultRouter
+from rest_framework.schemas import get_schema_view
+
+from hawc import __version__
+
+from . import api, views
+
+
+def get_admin_urlpatterns(open_api_patterns) -> List:
```

Comment:
not my favorite pattern (or best work), but this makes it relatively simple to pass a list of patterns to add to django routes which is determined by the admin setting.